docs/ci-services.md

```diff
@@ -0,0 +1,102 @@
+# CI Services
+
+The CI services map defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  CIServiceName: {scaffolder: opts => scaffold(opts), public: true}
+}
+```
+
+## Existing scaffolders
+There are already multiple CI service scaffolders [available for
+consumption](https://github.com/form8ion/awesome#continuous-integration-services).
+Should your workflow fit into those, feel free to integrate them directly. If
+not, you are welcome and encouraged to build more (and maybe consider
+submitting a PR with a link 😎)!
+
+## Config
+
+### keys: __string__
+
+Keys in the CI services map define the choices the user will be presented with
+when choosing their desired CI service. These have no functional impact on the
+scaffolder.
+
+example: `CIServiceName`
+
+### values: __object__
+
+Values in the CI services map define the actual mechanisms and configuration
+that will be used by the JavaScript Scaffolder to scaffold the CI service for
+the user's project.
+
+#### scaffolder: __function__ (_required_)
+
+Consumer-provided plugin called when this CI service is selected by the
+user in the interactive prompt. Will be called with the following arguments:
+
+* `projectRoot` __string__ full path to the directory where scaffolder will
+  create the JavaScript project
+* `vcs` __object__
+  * `host` __string__ VCS hosting service to generate the repo with (e.g., `github`)
+  * `owner` __string__ account name on the host service for the repository to
```

Comment:
```suggestion
  * `owner` __string__ account name on the host service to generate the repository
    within (e.g., `form8ion`)
```

Reply:
actually, js-scaffolder is not responsible for the generation of the repo. when using the project-scaffolder, the project-scaffolder does the repo generation. when using add-package-to-monorepo, the repo already exists

so, w/in the context of the js-scaffolder, the vcs details are simply informational for what repo the js-project details will be generated within

Reply:
this is making me wonder. how should we word this? given that it's the docs for the ci-services scaffolder. should we word it like the repo is already created? i could see both arguments but i'm not sure it's totally clear.

Reply:
i think we can leave whether the repo exists or not yet ambiguous. i'd try to focus on it being the repo within which the js stuff will end up, regardless of order/timeline